02-regex/README.md

```diff
@@ -87,6 +88,7 @@ _(Please note you are welcome to post under a pseudonym and/or password protect
 - Joyce -- [Week 2 - regex_replace_cat_with_dog](https://joycezheng.notion.site/assignment2-1024ee8df27e802fa70ec39c8899f505?pvs=4)
 - Seun Elemo -- [Punctuations are important! (Regex)](https://tinyurl.com/3pknd96d)
 - LovelyA72 -- [Week2](https://a2z.kmoene.com/p/393012-1718.html)
+- Lening --[https://replicate.com/zylim0702/remove-object?prediction=g20ykx9m6srgp0cf5e9ap3pygm]
```

Comment:
I wasn't sure what this link is referring to! Perhaps you meant this to go in the API section and we could talk about how to communicate with Replicate from p5.js?